### DISCUSSION TITLE
Error bindings models on routes using api

### DISCUSSION THREAD

**Discussion by felipechiodini** [laravel/framework#48093] [Q&A] [answered]
### Laravel Version

10.19.0

### PHP Version

8.1.22

### Database Driver & Version

MySql 8.1.0

### Description

Hello, 

I have an issue on my Laravel project, I use at my work 8.x laravel version with php 7.3.x, and use so much the resouce of binding the models directly on routes.

So I started an personal project using laravel lastest version and this resource is not working, I already made a deep search on google, but nothing found.

I tried to start new project and migrate the files, but still not working the binding.

next an image about my test, I do the most simple test to ensure that is not me doing something wrong.

this is the project: https://github.com/felipechiodini/hamburguer-api

![image](https://github.com/laravel/framework/assets/50850744/2d303057-1938-4be8-acc7-2183e33e798e)

### Steps To Reproduce

using api routes, try to bind an model directly on the route

**Comment by crynobone** [comment]
Moved to Discussions as this feels like a usage issue rather than a bug.

**Comment by henzeb** [comment]
and you're sure user with id 7 does exist and has an email?

**Reply by felipechiodini** [reply]
Yes, I sure, I checked everything also the middleware called on kernel, and everything looks like ok.

**Comment by felipechiodini** [comment, marked as answer]
I already discovered what's wrong, and It was my fault, when I change my RouteServiceProvider I don't notice and I omited the middleware.

![image](https://github.com/laravel/framework/assets/50850744/2b1affaa-ea93-4555-9e0f-a05954b9f69f)
